crates/rspack_plugin_javascript/src/visitors/mod.rs

```diff
@@ -75,12 +80,12 @@ pub fn run_before_pass(
       //      ),
       Optional::new(
         swc_visitor::decorator(assumptions, &options.builtins.decorator),
-        syntax.decorators()
+        transform_by_default && syntax.decorators()
```

Comment:
why not skip the whole transform_with_handler directly

Reply:
Still need to apply `DefinePlugin` etc.